controller/app/helpers/admin_helper.rb

```diff
@@ -99,8 +99,14 @@ def get_user_info(user)
     user["ssh_keys"].each { |k| user_ssh_keys << ["#{user['_id'].to_s}-#{k['name']}", Digest::MD5.hexdigest(k["content"]), nil] if k["content"] } if user["ssh_keys"].present?
     user_caps = user["capabilities"]
     if user["plan_id"]
-      plan_caps = OpenShift::BillingService.instance.get_plans[user["plan_id"].to_sym][:capabilities]
-      user_caps = plan_caps.deep_dup.merge(user_caps)
+      plans = OpenShift::BillingService.instance.get_plans[user["plan_id"].to_sym]
+      if plans.present?
+        plan_caps = plans[:capabilities]
+        user_caps = plan_caps.deep_dup.merge(user_caps)
+      else
+        puts "Warning: The plan_id: #{user["plan_id"]} exists, but does not have a valid plan.  Default user capabilities will be used."
+        puts "Has get_plans in the billing provider been implemented?"
```

Comment:
It looks like this warning is going to be displayed for every user. On a system with >5k users where all users have a plan_id but `get_plans` is not implemented, that is a lot of unnecessary output.

Maybe move the warning out of the loop and state something like the following:
```
Warning: At least one user has a plan id defined but no matching plans could be found. User capabilities defined by the plan will not be taken into consideration.
```